.github/workflows/integration-tests.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           key: ${{ runner.os }}-jdk-${{ matrix.java }}-maven-${{ hashFiles('**/pom.xml') }}
           restore-keys: |
             ${{ runner.os }}-jdk-${{ matrix.java }}-maven-
-      - run: mvn -B verify -Dtest=DebianITCase,DebianGpgITCase,BinaryITCase,RpmITCase
+      - run: mvn -B verify -Dit.test=DebianITCase,DebianGpgITCase,BinaryITCase,RpmITCase -Pitcase
```

Comment:
@olenagerasimova why do we need to specify integration tests to run? Failsafe plugin runs all tests ends with `ITcase` by default.

Reply:
@g4s8 same reason you have commented running ITs in this workflow at the beginning of the issue this PR is addressed to: we are remaking ITs and not all them are remade yet, some tests in the "old format" are failing.

Reply:
@olenagerasimova I see. In that case it could be better to disable the failing test with `@Disabled` annotation - it'll be easier to understand what tests needs to be fixed and the developer won't forget to add the test to CI configuration when fix it. WDYT?